skyportal/facility_apis/tess.py

```diff
@@ -98,16 +106,36 @@ def commit_photometry(lc, request_id, instrument_id, user_id):
 
         data_out = {
             'obj_id': request.obj.id,
+            'series_name': 'tesstransients',
+            'series_obj_id': request.obj.id,
```

Comment:
I should just mention that it may be more informative to put the TESS sector as the series name and the TIC ID as the obj_id, just so you can trace back where this data comes from. At least, that was the intention behind this field, but you can use it for anything that's relevant.